.Rbuildignore

```diff
@@ -51,3 +51,9 @@
 ^test_cases$
 ^test_test_cases\.R$
 
+^computing-environment_cache$
```

Comment:
what is this?

Reply:
Good catch. It's a leftover form another PR that didn't fly. I removed it.